src/exercises/day-3/afternoon.md

```diff
@@ -2,6 +2,14 @@
 
 Let us build a safe wrapper for reading directory content!
 
+For this exercise, we suggest attempting to use a local dev environment instead
+of using Playground if possible. This will allow you to attempt to run your
+binary on your own machine.
```

Comment:
I think we can make the text more direct here:

```suggestion
For this exercise, we suggest using a local dev environment instead
of the Playground. This will allow you to run your binary on your own machine.
```